integreat_cms/cms/forms/pages/page_filter_form.py

```diff
@@ -4,16 +4,42 @@
 from django.utils.translation import ugettext_lazy as _
 
 from ..custom_filter_form import CustomFilterForm
-from ...constants import translation_status
+from ...constants import translation_status, status
 
 logger = logging.getLogger(__name__)
 
+CHOICES = [("", "-----")] + list(status.CHOICES)
+
 
 class PageFilterForm(CustomFilterForm):
     """
     Form for filtering page objects
     """
 
+    status = forms.ChoiceField(
```

Comment:
I find it a bit weird that we use a `Select` widget for the publication status and a `CheckboxSelectMultiple` for the translation status - I would rather use one consistent method since both status-types are very similar.
In my mind, multiple checkboxes have the advantage that we can also select combinations (e.g. all translations in draft or review status) which would not be possible with a simple select field, but ok, I won't argue with this if other people are sure they want a select field there :sweat_smile:

Reply:
Thank you for your opinion. I certainly see your point. However @ulliholtgrave recommended to use the Select widget. I don't really have enough insight to decide this :/

Reply:
Alright, let's keep it that way and change it afterwards if necessary.